apps/site/src/middleware.page/return-types-as-json.ts

```diff
@@ -59,5 +59,6 @@ export const returnTypeAsJson = async (request: NextRequest) => {
     );
   }
 
-  return generateJsonResponse(type);
+  // @todo remove this cast when new type hosting available
+  return generateJsonResponse(type as DataType | EntityType | PropertyType);
 };
```

Comment:
@CiaranMn I had to add this to satisfy `tsc`. Considering we're merging `0.3` imminently this seems acceptable risk